### ISSUE TITLE
Cassandra v2.0.x incompatibility

### ISSUE THREAD

**Issue by RobinUS2** [prestodb/presto#7943]
Hi there, 

As of one of the latest versions (somewhere after 0.160, I think it's in 0.167 given https://prestodb.io/docs/current/release/release-0.167.html).

The change that causes this v2.0x incompatibility seems around https://github.com/prestodb/presto/commit/c7bca758548b5274be6ae529dbd201272a59d66f which forces the V3 protocol. 

The version we use in the shaded presto version is 3.1.4 https://github.com/prestodb/presto-cassandra-driver/blob/master/pom.xml#L46 and according to datastax this is compatible with the 2.0.x branch (see  https://docs.datastax.com/en/developer/java-driver/3.1/ ).

Can we make this configurable in the catalog? E.g. `cassandra.protocol-version=V2`

Happy to implement the changes myself.

**Comment by cawallin** [comment]
Yeah, that should be fine -- there was just a problem with the driver defaulting to a newer driver that was causing intermittent connection issues in tests. I'll review the PR now.

**Comment by electrum** [comment]
If this is good, I'm fine to merge for 0.175
